webrepl.html

```diff
@@ -17,6 +17,10 @@
     font: 20px/1.5 sans-serif;
   }
 
+  #outer {
+    position: fixed;
+  }
+
 /*
   .terminal {
```

Comment:
If this section is commented in it would be possible to override the font-size (and the other properties) from the html rather than using the default ones from the terminal.js file (see my pull request). The terminal.js also uses 11px (pixels) as default font size - maybe somehting resolution independant is better? E.g. "14pt", or "small", "medium"